pkg/reconciler/route/table_test.go

```diff
@@ -2629,6 +2629,7 @@ func ReconcilerTestConfig(enableAutoTLS bool) *config.Config {
 			AutoTLS:                    enableAutoTLS,
 			DomainTemplate:             network.DefaultDomainTemplate,
 			TagTemplate:                network.DefaultTagTemplate,
+			HTTPProtocol:               network.HTTPEnabled,
```

Comment:
Can we add some test cases with `httpProtocol: Disabled` or `Redirected`?

Reply:
Thank you. Yes, I added it now.